src/context/AppProvider/AppContext.tsx

```diff
@@ -168,24 +163,11 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
       if (portfolioLoadingStatus === 'loading') return
 
       const { getFoxyRebaseHistoryByAccountId } = txHistoryApi.endpoints
-      const { getValidatorData } = validatorDataApi.endpoints
 
       // forceRefetch is enabled here to make sure that we always have the latest state from chain
       // and ensure the queryFn runs resulting in dispatches occuring to update client state
       const options = { forceRefetch: true }
 
-      // Sneaky hack to fetch cosmos SDK default opportunities for wallets that don't support Cosmos SDK
-      // We only store the validator data for these and don't actually store them in portfolio.accounts.byId[accountId].stakingDataByValidatorId
-      // Since the accountId is an empty address (generated and private keys burned) and isn't actually in state
-      if (wallet && !supportsCosmos(wallet)) {
-        const accountId = `${cosmosChainId}:${EMPTY_COSMOS_ADDRESS}`
-        dispatch(getValidatorData.initiate(accountId, options))
-      }
-      if (wallet && !supportsOsmosis(wallet)) {
-        const accountId = `${osmosisChainId}:${EMPTY_OSMOSIS_ADDRESS}`
-        dispatch(getValidatorData.initiate(accountId, options))
-      }
-
```

Comment:
This should show no regressions since we now always fetch the two default validators metadata, and don't need an `AccountId` for that, but regression test me

Reply:
Confirmed no regressions, tested with the testing steps of https://github.com/shapeshift/web/pull/2545

<img width="1147" alt="image" src="https://user-images.githubusercontent.com/17035424/217371798-1feddcc0-2e45-4c81-b89f-89fb04583d40.png">

Keeping this conversation open for discoverability, resolve me after you've read me